tex/homology/singular.tex

```diff
@@ -666,7 +666,7 @@ \section{More examples of chain complexes}
 	$[v_0,\ldots,v_n]$, we remove each vertex one-by-one, and take
 	the alternating sum. Therefore, $\partial([v]) = []$. Extending
 	it linearly to complexes yields
-	$\partial(\sum n_ip_i) = \sum n_i \cdot 1$ --- so $\epsilon$ really is
+	$\partial(\sum n_ip_i) = \sum n_i \cdot 1$ --- so $\eps$ really is
```

Comment:
For consistency --- everywhere else in the book either \eps or \varepsilon is used (which renders the same)